source/includes/administration/_additional_roles.md

```diff
@@ -0,0 +1,115 @@
+
+`GET /users/:id/additional_roles`
+
+```shell
+# Update an environment
+curl -X GET "https://cloudmc_endpoint/v2/users/:id/additional_roles" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json"
+
+# Response body example
+```
+```json
+{
+"data": [
+    {
+        "id": "ed18cdd0-0fa0-464d-a993-e689d39dfe2e",
+        "creationDate": "2019-06-06T15:01:22.000Z",
+        "scopeQualifier": "ORG_BASE",
+        "role": {
+            "id": "cc71a40b-d90b-40db-b724-08d71f5179b5",
+            "name": "user",
+            "isSystem": true,
+            "isFixed": true
+        },
+        "organization": {
+            "name": "Test",
+            "id": "6861c664-0a83-4e91-bd8c-3a6083e81a13",
+            "entryPoint": "test"
+        },
+        "user": {
+            "id": "6ebc54bb-c3fe-4434-b412-09ac4f798680",
+            "firstName": "franz",
+            "lastName": "franz",
+            "userName": "franz",
+            "email": "fgarcia@cloudops.com"
+        }
+    }]
+}
+```
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the additional role
+`creationDate`<br/>*string* | The date the additional role was added/updated on the user
+`scopeQualifier`<br/>*string* | The scope of this additional role (ORG_BASE, TAGS_ANYMATCH, GLOBAL, etc..). Please refer to the table above.
+`tags`<br/>*[string]* | Organization tags to match for TAGS_ANYMATCH scoping
+`role`<br/>*[Role](#administration-roles)* | The role to apply on the target scope
+`organization`<br/>*[Organization](#administration-organizations)* | The organization of the additional role. Only present on scopes that target a specific organization.<br/>*includes*: `id`, `name`, `entryPoint`
+`user`<br/>*[User](#administration-users)* | The user that the additional role is applied on<br/>*includes*: `id`, `username`, `firstName`, `lastName`, `email`
+
+
+### Assign additional role to user
+
+`POST /users/:id/additional_roles`
+
+```shell
+# Update an environment
+curl -X POST "https://cloudmc_endpoint/v2/users/:id/additional_roles" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json"
+   -d "[request_body]"
+
+# Request body example
+```
+```json
+{
+    "scopeQualifier": "ORG_BASE",
+    "role": {
+        "id": "cc71a40b-d90b-40db-b724-08d71f5179b5",
+    },
+    "organization": {
+        "id": "6861c664-0a83-4e91-bd8c-3a6083e81a13",
+    },
+}
+```
+
+Required | &nbsp;
+---------- | -----------
+`scopeQualifier`<br/>*string* | The scope of this additional role (ORG_BASE, TAGS_ANYMATCH, GLOBAL, etc..)
+`role.id`<br/>*UUID* | The id of the role to apply on the target scope
+`organization.id`<br/>*UUID* | The id of the target organization of the scope. Only required when using a scope qualifier that targets an organization.
+`tags`<br/>*UUID* | The organization tags of the scope. Only required when using the TAGS_ANYMATCH scope.
+
+You will need to have `Admin users` permission to execute this operation.
```

Comment:
How should we present the permissions names?:
1. `Admin users`
2. `Admin:users`
3. `Manage users`

Reply:
good question ;S

Reply:
I think it should be the user label.
![image](https://user-images.githubusercontent.com/6817637/59102003-c632dd80-88f8-11e9-80ba-971a394b55d1.png)

so:
4. `Users : Manage`

Reply:
@mavailla just confirmed it should be the user label.

Reply:
+1

Reply:
Fixed